learningCodableJSON/DataCity.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DataCityTableViewCell.swift
+//  learningCodableJSON
+//
+//  Created by Larissa Diniz  on 11/02/20.
+//  Copyright © 2020 Larissa Diniz . All rights reserved.
+//
+
+import UIKit
+
+class DataCity: UIViewController {
+
+    @IBOutlet weak var cityLabel: UILabel!
+    @IBOutlet weak var nameCityLabel: UILabel!
+    @IBOutlet weak var humidityLabel: UILabel!
+    @IBOutlet weak var valueHumidityLabel: UILabel!
+    @IBOutlet weak var minLabel: UILabel!
+    @IBOutlet weak var valueMinLabel: UILabel!
+    @IBOutlet weak var maxLabel: UILabel!
+    @IBOutlet weak var valueMaxLabel: UILabel!
+    
+    var weather: Weather?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        nameCityLabel.text = weather?.city
+        valueHumidityLabel.text = "\(weather!.humidity)%"
```

Comment:
Remove esses force unwrap.
Se por um acaso esse dado estiver faltando, teu app quebra

Reply:
Quando eu deixo como opcional isso acontece:

![Screen Shot 2020-02-14 at 14 10 30](https://user-images.githubusercontent.com/55598696/74552342-c6798300-4f33-11ea-9410-4f11df2d3fe2.png)

E quando eu forço ele mostra assim: 

![Screen Shot 2020-02-14 at 14 12 52](https://user-images.githubusercontent.com/55598696/74552494-1b1cfe00-4f34-11ea-96bd-917bcdc79261.png)